src/commands/add/addCredential.ts

```diff
@@ -162,7 +162,9 @@ export const getTokens = async (
     target
   ).catch((e) => {
     process.logger?.error(
-      `An error has occurred while validating your credentials: ${e}\nPlease check your Client ID ${
+      `An error has occurred while validating your credentials: `,
```

Comment:
you should concatenate these string because `process.logger.error` expect 1st argument to be a message